### PR TITLE
feat(angular): upgrade to rc4

### PR DESCRIPTION
**Pull request by wesleycho**
- Upgrade angular to rc4

### CODE REVIEW

File: demo/src/main.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { enableProdMode } from '@angular/core';
 import { bootstrap } from '@angular/platform-browser-dynamic';
-import { ELEMENT_PROBE_PROVIDERS } from '@angular/platform-browser';
+import { ELEMENT_PROBE_PROVIDERS } from '@angular/platform-browser/src/dom/debug/ng_probe';
```
Comment (pkozlowski-opensource) [inline demo/src/main.ts]: This sounds bad... Why do we need `ELEMENT_PROBE_PROVIDERS`?

Comment (wesleycho) [inline demo/src/main.ts]: No clue - @Foxandxss, do you know the reason? Removing the logic around here doesn't seem to break anything.

Comment (Foxandxss) [inline demo/src/main.ts]: That is a good question, I carried that over from my starter. Fire with it.

**Comment by Foxandxss** [comment]
I merged this by  hand.

**Pull request closed without merging**